deploy.sh

```diff
@@ -30,8 +30,8 @@ cp -rf ${CWD_DIR}/bin/*.a    ${DEPLOY_BUILD_HOME}/lib/
 cp -rf ${CWD_DIR}/bin/*.so   ${DEPLOY_BUILD_HOME}/lib/
 cp -rf ${CWD_DIR}/include    ${DEPLOY_BUILD_HOME}/
 cp -rf ${CWD_DIR}/example    ${DEPLOY_BUILD_HOME}/
-cp -rf ${CWD_DIR}/doc 	     ${DEPLOY_BUILD_HOME}/
-cp -rf ${CWD_DIR}/changelog  ${DEPLOY_BUILD_HOME}/
+#cp -rf ${CWD_DIR}/doc 	     ${DEPLOY_BUILD_HOME}/
+#cp -rf ${CWD_DIR}/changelog  ${DEPLOY_BUILD_HOME}/
```

Comment:
Could you please tell the reason for commenting these lines?
If the `${CWD_DIR}/doc` is not existed, I think we should fix the issue that `${CWD_DIR}/doc` could not be found instead of commenting these directly?